pkgs/by-name/xe/xephem/package.nix

```diff
@@ -0,0 +1,83 @@
+{
+  lib,
+  gcc13Stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
+  copyDesktopItems,
+  installShellFiles,
+  motif,
+  openssl,
+  groff,
+  xorg,
+}:
+
+gcc13Stdenv.mkDerivation rec {
```

Comment:
Please add a comment why we use GCC 13 here.

If it is just due to the new warnings-made-errors, I would suggest to set env.NIX_CFLAGS_COMPILE = "-Wno-error=..." instead.

Reply:
Simple, that fix is not in the released version 4.2.0 https://github.com/XEphem/XEphem/commit/30e14f685ede015fcd8985cd83ee6510f93f0073

Add `patches = [ (fetchpatch { url = "https://github.com/XEphem/XEphem/commit/30e14f685ede015fcd8985cd83ee6510f93f0073.patch"; hash = ""; }) ]`. (Build and copy the hash you get.)